suites/nautilus/cephfs/tier_2_cephfs_test-snapshot-clone.yaml

```diff
@@ -0,0 +1,156 @@
+#	CEPH-83573521 Remove a subvolume group by retaining the snapshot : ceph fs subvolume rm <vol_n...
+#	CEPH-83573415 Test to validate the cli - ceph fs set <fs_name> allow_new_snaps true
+#	CEPH-83573418 Create a Snapshot, reboot the node and rollback the snapshot
+#	CEPH-83573420 Try writing the data to snap directory
+#   CEPH-83573524	Ensure the subvolume attributes are retained post clone operations
+#   CEPH-11319		Create first snap add more data to original then create a second snap.
+#                   Rollback 1st snap do data validation. Rollback 2nd snap and do data validation.
+#                   Perform cross platform rollback i.e. take snap on kernel mount and perform rollback using fuse mount
+#   CEPH-83573255	Try renaming the snapshot directory and rollbackCreate a FS and
+#                   create 10 directories and mount them on kernel client and fuse client(5 mounts each)
+#                   Add data (~ 100 GB). Create a Snapshot and verify the content in snap directory.
+#                   Try modifying the snapshot name.
+#   CEPH-83573522	Verify the retained snapshot details with "ceph fs info" command
+#===============================================================================================
+tests:
+  -
+    test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: "install ceph pre-requisites"
+  -
+    test:
+      abort-on-fail: true
+      config:
+        ansi_config:
+          alertmanager_container_image: "registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6"
+          ceph_conf_overrides:
+            client:
+              rgw crypt require ssl: false
+              rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+                testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+            global:
+              mon_max_pg_per_osd: 1024
+              osd_default_pool_size: 2
+              osd_pool_default_pg_num: 64
+              osd_pool_default_pgp_num: 64
+            mon:
+              mon_allow_pool_delete: true
+          ceph_docker_image: rhceph/rhceph-4-rhel8
+          ceph_docker_image_tag: latest
+          ceph_docker_registry: registry.redhat.io
+          ceph_origin: distro
+          ceph_repository: rhcs
+          ceph_stable: true
+          ceph_stable_release: nautilus
+          ceph_stable_rh_storage: true
+          ceph_test: true
+          cephfs_pools:
+            -
+              name: cephfs_data
+              pgs: "8"
+            -
+              name: cephfs_metadata
+              pgs: "8"
+          containerized_deployment: true
+          copy_admin_key: true
+          dashboard_admin_password: p@ssw0rd
+          dashboard_admin_user: admin
+          dashboard_enabled: true
+          grafana_admin_password: p@ssw0rd
+          grafana_admin_user: admin
+          grafana_container_image: "registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4"
+          journal_size: 1024
```

Comment:
If required, could you add in the docstring about the environment specific requirement.